### ISSUE TITLE
Android build error after Huawei integration

### ISSUE THREAD

**Issue by malwatte** [Pushwoosh/pushwoosh-react-native-plugin#145]
Getting this error when running Android build after Huawei integration.

```
--W- The variant: debug, Use the json file: /Users/***/new/HuaweiConfigApp/android/app/agconnect-services.json
FAILURE: Build failed with an exception.
* Where:
Build file '/Users/***/HuaweiConfigApp/node_modules/pushwoosh-react-native-plugin/src/android/build.gradle' line: 43
* What went wrong:
A problem occurred evaluating project ':pushwoosh-react-native-plugin'.
> Circular referencing during evaluation for project ':app'.
```

**Steps to reproduce:**
1. We use react-native 0.68.x. So use this to create a new project.
`npx react-native init HuaweiConfigApp --template react-native-template-typescript@6.10.2`

2. Integrate Pushwoosh for Android by following this docs
https://docs.pushwoosh.com/platform-docs/pushwoosh-sdk/cross-platform-frameworks/react-native/integrating-react-native-plugin

3. Clean & build the app
`./gradlew clean`
`./gradlew assembleDebug`
Success

4. Integrate Huawei push notifications by following this doc
https://docs.pushwoosh.com/platform-docs/pushwoosh-sdk/cross-platform-frameworks/react-native/integrate-huawei-react-native

5. Clean & build the app
`./gradlew clean`
ERROR (see above mentioned error)

Appreciate if someone can help.

**Comment by malwatte** [comment]
Hi,
Any update?

**Comment by wfhm** [comment]
@malwatte checking it

**Comment by wfhm** [comment]
@malwatte Fixed in 6.1.17.
